models/issue_label.go

```diff
@@ -43,7 +70,7 @@ func GetLabelTemplateFile(name string) ([][3]string, error) {
 			return nil, fmt.Errorf("line is malformed: %s", line)
 		}
 
-		if !labelColorPattern.MatchString(fields[0]) {
+		if !LabelColorPattern.MatchString(fields[0]) {
```

Comment:
Maybe if we have `#` optional in the API it should be optional everywhere?

Reply:
and beside the api where do you mean?

Reply:
Resource files.

Reply:
custom/.../labels/{set}

Reply:
I'll have a look at it